src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,105 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static User user;
+    private static RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
+
+    @BeforeAll
+    static void beforeAll() {
+        User user1 = new User(1L, "alex123", "password1", 25);
+        User user2 = new User(2L, "johnDoe", "123456", 30);
+        User user3 = new User(3L, "tester99", "qwerty", 19);
+        User user4 = new User(4L, "maria87", "securepass", 28);
+        User user5 = new User(5L, "bobsmith", "mypwd789", 35);
+        registrationService.register(user1);
+        registrationService.register(user2);
+        registrationService.register(user3);
+        registrationService.register(user4);
+        registrationService.register(user5);
```

Comment:
don't use the method register before all you can’t be sure that this method works correctly. Add user directly using `storageDao.add(user)`